doc/readme.rst

```diff
@@ -0,0 +1,134 @@
+.. figure:: luigi.png
```

Comment:
they can, yes.
I don't see an easy alternative to have both in sync though due to the external image problem though.

maybe they actually should be different things and have different content?
simplify the README github page (ala https://github.com/ansible/ansible)?
make a more in-depth intro on the readthedocs (just like it is)?